data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/SqsService.java

```diff
@@ -14,6 +15,9 @@
 
 public class SqsService {
     private static final Logger LOG = LoggerFactory.getLogger(SqsService.class);
+    static final int INITIAL_DELAY = 1000;
+    static final int MAXIMUM_DELAY = 30 * 60 * 1000;
```

Comment:
30 minutes seems rather long. I think 5 minutes as a maximum might be better.

Reply:
But, I'm fine with either. I'm good with this change once you address the other PR feedback.